swagger/wrhs-status-api-spec.json

```diff
@@ -2,7 +2,7 @@
   "openapi": "3.0.2",
   "info": {
     "title": "warehouse.ai-status-api",
-    "version": "1.0.1",
+    "version": "1.1.1",
```

Comment:
You should change it to not commit this file and just have it setup in pretest/prepare scripts like we're starting to do in other repos.